### ISSUE TITLE
hardware_info: different fields between native_python and raspberry

### ISSUE THREAD

**Issue by cstenglein** [fusion44/blitz_api#80]
The fields from hardware_info are different between the implementations:

https://github.com/fusion44/blitz_api/tree/main/app/repositories/hardware_impl

e.g. native_python has `disk_io` stuff which would be cool

**Comment by fusion44** [comment]
I agree. On RaspiBlitz we only fetch the available hardware values that are available through Redis instead of fetching them with Python for performance reasons. @rootzoll what do you think? Should I fetch them via python or do we add more fields via the native RaspiBlitz script?

**Comment by rootzoll** [comment]
> what do you think? Should I fetch them via python or do we add more fields via the native RaspiBlitz script?

If its not a bit performance think and its just system info data you can also use the python native way.
